docs/mkdocs/docs/development.md

```
@@ -0,0 +1,380 @@
+
+#### Best Practices
+
+- Do not use `And`, use `Given`, `When` or `Then` explicitly.
+  - This helps avoid unexpected behavior when commenting out steps.
+
+### Tests Generation
+
+You can use [GitHub.copilot] to generate tests boiler plate using prompts like following:
+
+> _Generate gherkin test for {filename}._
+
```

Markdownlint (0.37.0)

Blank line inside blockquote

(MD028, no-blanks-blockquote)